db/active_replicator_checkpointer_test.go

```diff
@@ -249,6 +252,15 @@ func TestCheckpointerSafeSeq(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			stats, err := base.SyncGatewayStats.NewDBStats(t.Name(), false, false, false, nil, nil)
+			replicationStats, err := stats.DBReplicatorStats(t.Name())
+			require.NoError(t, err)
+			replicatorConfig := &ActiveReplicatorConfig{}
+			replicatorConfig.ReplicationStatsMap = replicationStats
+			tt.c.stats.ProcessedSequenceLen = replicatorConfig.ReplicationStatsMap.ProcessedSequenceLen
+			tt.c.stats.ProcessedSequenceLenPostCleanup = replicatorConfig.ReplicationStatsMap.ProcessedSequenceLenPostCleanup
+			tt.c.stats.ExpectedSequenceLen = replicatorConfig.ReplicationStatsMap.ExpectedSequenceLen
+			tt.c.stats.ExpectedSequenceLenPostCleanup = replicatorConfig.ReplicationStatsMap.ExpectedSequenceLenPostCleanup
```

Comment:
Could be simplified a bit more to this, since we don't care about Prometheus or any other NewDBStats/DBReplicatorStats stuff in this particular test.

Would still allow us to assert on these stats using something like `tt.c.stats.ExpectedSequenceLen.Value()` inside the test

```suggestion
			tt.c.stats = CheckpointerStats{
				ExpectedSequenceLen:             &base.SgwIntStat{},
				ExpectedSequenceLenPostCleanup:  &base.SgwIntStat{},
				ProcessedSequenceLen:            &base.SgwIntStat{},
				ProcessedSequenceLenPostCleanup: &base.SgwIntStat{},
			}
```